docs/deployment/upgrade.md

```diff
@@ -8,53 +8,60 @@ hide_title: true
+Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (e.g., 1.0.0). We use Lerna to manage versioning.
 
-To view the released module artifacts for each Cumulus core version, see the [Cumulus releases page](https://github.com/nasa/cumulus/releases).
+To view the released module artifacts for each Cumulus core version, see the [Cumulus Releases] page.
 
-**IMPORTANT:** Cumulus releases may introduce breaking changes between minor versions (e.g. 1.13.x -> 1.14.0), so make sure to consult the release notes for migration steps.
+**IMPORTANT:** Cumulus releases may introduce breaking changes between minor versions (e.g., 1.13.x -> 1.14.0), so it is critical that you consult the release notes for migration steps.
 
-## Migrating to a new version
+## Migrating to a New Version
 
-When breaking changes have been introduced, the Cumulus Core team will publish instructions on migrating from one version to another.
+When breaking changes have been introduced, the Cumulus Core team will publish instructions on migrating from one version to another.  Detailed release notes with migration instructions (if any) for each release can be found on the [Cumulus Releases] page.
 
-Detailed release notes with migration instructions (if any) for each release can be found on the [Cumulus releases page](https://github.com/nasa/cumulus/releases).
+**IMPORTANT** 
 
-**IMPORTANT:** When upgrading through many versions, each migration should be done in the order of release (if going from version 1.1.0 from 1.3.0, upgrade from 1.1.0 to 1.2.0 and then to 1.3.0).
+1. **Use consistent Cumulus versions:** All modules must be updated to the same Cumulus version number (see below)
+2. **Follow all intervening steps:** When skipping over versions, you **must perform all intervening migration steps**.  For example, if going from version 1.1.0 to 1.3.0, upgrade from 1.1.0 to 1.2.0 and then to 1.3.0.  This is critical because each release that contains migration steps provide instructions _only_ for migrating from the _immediately_ previous release, but you must follow _all_ migration steps between your currently installed release and _every release_ through the release that you wish to migrate to.
+3. **Migrate lower environments first:** Migrate your "lowest" environment first and test it to ensure correctness before performing migration steps in each successively higher environment.  For example, in the EDCloud progress from a sandbox environment to UAT to SIT to PROD.
+4. **Migrate during appropriate times:** Choose a time to migrate when support is more likely to be available in case you encounter problems, such as when you are most likely to quickly find help in the #cumulus-internal Slack channel.  Prefer earlier in the week over later in the week (particularly avoiding Fridays, if possible).
```

Comment:
```suggestion
4. **Migrate during appropriate times:** Choose a time to migrate when support is more likely to be available in case you encounter problems, such as when you are most likely to be able to obtain support.  Prefer earlier in the week over later in the week (particularly avoiding Fridays, if possible).
```

Again, generic audience concern.    Perhaps more importantly that location might change in the future, and I'm certain we won't remember to update this. 

 *Probably* not a big deal, but if we start seeing issues from external entities asking where our slack channel is and how they get support.... :D

Reply:
Yep, agree. We should leave the Slack channel out of this docs.

Reply:
Makes sense.